AdvancedStateAndSideEffectsCodelab/app/src/androidTest/java/androidx/compose/samples/crane/home/HomeTest.kt

```diff
@@ -35,13 +35,6 @@ class HomeTest {
     @get:Rule(order = 1)
     val composeTestRule = createAndroidComposeRule<MainActivity>()
 
-    @Before
```

Comment:
Warning here https://developer.android.com/jetpack/compose/testing

Reply:
cc @JolandaVerhoef do we need to update the guidance? 

java.lang.IllegalStateException: androidx.compose.samples.crane.home.MainActivity@7a9ade8 has already set content. If you have populated the Activity with a ComposeView, make sure to call setContent on that ComposeView instead of on the test rule; and make sure that that call to `setContent {}` is done after the ComposeTestRule has run

Reply:
I think this was just a bug in Crane that has since been fixed in the Compose Samples Crane implementation. Combining a custom Activity (that sets its own Compose app content) and then explicitly calling setContent again makes no sense from a testing perspective.